src/components/LanguageDropdown/LanguageDropdown.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ReactComponent as LanguagePicker } from './LanguagePicker.svg';
+import { languageOptions } from '../../data/LanguageContent';
+
+import './LanguageDropdown.css';
+
+const LanguageDropdown = ({ language, setLanguage }) => {
+  return (
+    <div className='languageDropdown'>
+      <div className='label'>
+        <LanguagePicker height='2rem' width='2rem' />
+        <h4>{ language.language }:</h4>
+      </div>
+      <div>
+        {/* <LanguagePicker height='2rem' width='2rem' /> */}
```

Comment:
can we remove the commented out code? with source control really no need to ever leave in comments unless for like a "future feature" but rarely then as well